### ISSUE TITLE
Errors, but no errors

### ISSUE THREAD

**Issue by lagden** [sindresorhus/xo#84]
Hi,

I've got this problem using `xo global version`

<img width="922" alt="screen shot 2016-03-14 at 12 44 52" src="https://cloud.githubusercontent.com/assets/130963/13749965/4c2de90c-e9e2-11e5-87d7-5bec3e5e3ce9.png">

Probably solution: https://github.com/airbnb/javascript/issues/728

```
node v5.6.0
xo v0.13.0
```

**Comment by lagden** [comment]
I left with this settings to avoid errors.

```json
{
  "xo": {
    "rules": {
      "no-empty-label": [0],
      "space-before-keywords": [0],
      "space-after-keywords": [0],
      "space-return-throw-case": [0]
    }
  }
}
```
 workaround!! :sweat_smile:

**Comment by sindresorhus** [comment]
Make sure your local XO is also updated. This was fixed in XO 0.13.0. So should not happen.

Can you give me the output of `$ npm ls`?

**Comment by lagden** [comment]
I found the problem!

```
$ npm ls | ack xo
│ │ │ └── indexof@0.0.1
├── eslint-config-xo@0.6.0 extraneous <--- HERE
├── gulp-xo@0.9.0
└─┬ xo@0.13.0
  ├── eslint-config-xo@0.12.0 
  └── xo-init@0.3.6
```

Thanks for helping! Fixed! :relaxed:
